engineio/README.md

```diff
@@ -55,3 +55,16 @@ as custom headers for the opening request.
 ## Documentation
 
 Documentation of this crate can be found up on [docs.rs](https://docs.rs/rust_engineio).
+
+## Async version
+
+The crate also ships with an asynchronous version that can be enabled with a feature flag.
+The async version implements the same features mentioned above.
+The asynchronous version has a similar API, just with async functions. Currently the futures
+can only be executed with [`tokio`](https://tokio.rs). In the first benchmarks the async version
```

Comment:
nit: ideally we wouldn't be tied to tokio. What are we doing that's dependent on tokio?

Reply:
Basically tokio-tungstenite ;) There is an `async-tungstenite` crate, but it needs an existing stream to abstract over. Might be something for the future, but for now I would settle with tokio. This crate is not capable of running in a `#[no_td]` environment, so being generic for use with the `smol` executor doesn't make sense.

Reply:
Alright, ideally eventually we won't require tokio. It's concerning how many crates depend on tokio being the runtime explicitly.

Reply:
True, but it's quite difficult to not be tied to an executor, especially for a networking library. I would go with tokio for now with the aim of changing this in the future.